patches/chrome-browser-resources-settings-route.js.patch

```diff
@@ -1,14 +1,21 @@
 diff --git a/chrome/browser/resources/settings/route.js b/chrome/browser/resources/settings/route.js
-index fe6ea7395fda29aa09c8caf03a063bdd5b7e7d2c..50f87646b0e456449c3cf56ca286a575f8702af5 100644
+index fe6ea7395fda29aa09c8caf03a063bdd5b7e7d2c..aad0b409b97bc0849a5f2d4e79cb4baef4496efa 100644
 --- a/chrome/browser/resources/settings/route.js
 +++ b/chrome/browser/resources/settings/route.js
-@@ -39,7 +39,9 @@ function addPrivacyChildRoutes(r) {
+@@ -7,7 +7,7 @@ import {loadTimeData} from 'chrome://resources/js/load_time_data.m.js';
+ import {pageVisibility} from './page_visibility.js';
+ import {Route, Router} from './router.js';
+ import {SettingsRoutes} from './settings_routes.js';
+-
++import addBraveRoutes from './brave_routes.js'
+ /**
+  * Add all of the child routes that originate from the privacy route,
+  * regardless of whether the privacy section under basic or advanced.
+@@ -214,6 +214,7 @@ function createBrowserSettingsRoutes() {
+       // </if>
+     }
+   }
++  addBraveRoutes(r)
```

Comment:
can we import in this same line?

Reply:
cc @petemill

Reply:
Doesn't seem like we can have import on the same line:
<pre>
[!] Error: 'import' and 'export' may only appear at the top level
</pre>